.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce
+
+### Minimal yet complete reproducer code (or GitHub URL to code)
+
+### SOFARPC version
```

Comment:
You can aggregate 3 version to `Environment`. 

like:https://github.com/alibaba/pouch/blob/master/.github/ISSUE_TEMPLATE.md